src/App.tsx

```diff
@@ -15,36 +23,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const getPreparedGoods = (
+  goods: string[],
+  sortType: SortType | null,
+  reverse: boolean,
+): string[] => {
+  const preparedGoods = [...goods];
+
+  if (sortType === SortType.Alphabetical) {
+    preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+  } else if (sortType === SortType.Length) {
+    preparedGoods.sort((good1, good2) => good1.length - good2.length);
+  }
```

Comment:
you check different values of sortType so better to use a switch case here instead of if/else